packages/turbo-vsc/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Can we simplify this and remove the comments?

Reply:
I am just going to wipe it and use the one in the root.